app/models/local_photo.rb

```diff
@@ -394,4 +467,112 @@ def extrapolate_dimensions_from_original
     end
   end
 
+  def s3_client
+    s3_credentials = LocalPhoto.new.file.s3_credentials
+    ::Aws::S3::Client.new(
+      access_key_id: s3_credentials[:access_key_id],
+      secret_access_key: s3_credentials[:secret_access_key],
+      region: LocalPhoto.s3_region
+    )
+  end
+
+  private
+
+  def self.move_to_appropriate_bucket( photo )
+    return unless CONFIG.s3_public_bucket
+    return unless p = LocalPhoto.find_by_id( p ) unless photo.is_a?( LocalPhoto )
```

Comment:
What is `p` here? I'm surprised this runs with the two `unless` statements on one line.

Reply:
This was just me flubbing a refactor of this method. I think I've fixed this now